tests/Bridge/Symfony/Bundle/Command/SwaggerCommandTest.php

```diff
@@ -11,27 +11,26 @@
 
 namespace ApiPlatform\Core\Tests\Bridge\Symfony\Bundle\Command;
 
-use Nelmio\ApiDocBundle\Tests\WebTestCase;
 use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
 use Symfony\Component\Console\Tester\ApplicationTester;
 
 /**
  * @author Amrouche Hamza <hamza.simperfit@gmail.com>
  */
-class SwaggerComandTest extends WebTestCase
+class SwaggerCommandTest extends KernelTestCase
```

Comment:
There is https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Console/Tester/CommandTester.php